source/extensions/tracers/opentelemetry/resource_detectors/environment/BUILD

```diff
@@ -0,0 +1,31 @@
+load(
```

Comment:
The idea I had is to allow each vendor/interested party to add their custom detectors here. 

```
resource_detectors
  - environment
  - dynatrace
  - aws
  - azure
  - gcp
  -...
```

Reply:
Ah yes, each would be under their own directory. Sorry that wasn't clear from my comment.